src/router.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import HomeContainer from "./containers/home";
+import ApolloClient from "apollo-boost";
+import { ApolloProvider } from "@apollo/react-hooks";
+
+// TODO: Move uri value to config file
+const client = new ApolloClient({
+  uri: "https://hardcore-haibt-da9a6b.netlify.com/.netlify/functions/graphql"
```

Comment:
I think this needs to be pulled into an environment variable so we can dynamically set the url based on the environment. 

i.e if node env is `development` then set this to `http://localhost:8888/.netlify/functions/graphql`